doc-checking-app-api/src/main/java/uk/gov/di/authentication/app/entity/DocAppAuthorisationResponse.java

```diff
@@ -1,10 +1,14 @@
 package uk.gov.di.authentication.app.entity;
 
 import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
 
 public class DocAppAuthorisationResponse {
 
     @JsonProperty("redirectUri")
```

Comment:
Do we still need this?

Reply:
Possibly not, there were a few places that I left both in because of the way the tests were run.

I'll be raising another PR later to totally remove Jackson from the project, I'll pick these remnants up when I do that.

Reply:
👍 cool, that makes sense.